.github/dependabot.yml

```diff
@@ -20,28 +20,3 @@ updates:
   schedule:
     interval: weekly
   open-pull-requests-limit: 1000
-- package-ecosystem: pip
-  directory: docs/
-  schedule:
-    interval: monthly
-  open-pull-requests-limit: 1000
-- package-ecosystem: pip
-  directory: resources/test/
-  schedule:
-    interval: monthly
-  open-pull-requests-limit: 1000
-- package-ecosystem: pip
-  directory: tools/
-  schedule:
-    interval: monthly
-  open-pull-requests-limit: 1000
-- package-ecosystem: pip
-  directory: tools/wpt/
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 1000
-- package-ecosystem: pip
-  directory: tools/wptrunner/
```

Comment:
I think we should just remove this rule, but otherwise not revert the patch; it seems dependabot works recursively anyway.